recipes/backward-cpp/all/conanfile.py

```diff
@@ -95,12 +106,18 @@ def _configure_cmake(self):
         cmake.definitions['STACK_DETAILS_BFD'] = self._has_stack_details("bfd")
         cmake.definitions['STACK_DETAILS_DWARF'] = self._has_stack_details("dwarf")
         cmake.definitions['BACKWARD_SHARED'] = self.options.shared
+        cmake.definitions['BACKWARD_TESTS'] = False
         cmake.configure(build_folder=self._build_subfolder)
         return cmake
 
+    def patch_sources(self):
```

Comment:
custom methods should be "protected"
```suggestion
    def _patch_sources(self):
```